lib/diaspora_federation/schemas/federation_entities.json

```diff
@@ -0,0 +1,373 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://diaspora.github.io/diaspora_federation/schemas/federation_entities-0.2.0-dev.json",
```

Comment:
(sorry I took so long, was busy with other stuff.)

My 👍 for using GitHub pages here. There is no real user impact if the site is unavailable and chances are low that we'll ever run into security issues on GitHub. Even if we do, we have local copies of the repos anyway. Also, updating the schemas is much easier on GitHub, since we just have to merge a PR. No reason to add a deploy chain to host it on diasporafoundation.org needed.